src/lib/resource/sections/task-list.js

```diff
@@ -56,14 +56,6 @@ module.exports = {
                                             nonDeceased: 'Details of the person acting on behalf of the victim'
                                         }
                                     }
-                                },
-                                't_rep_details': {
-                                    title: {
-                                        proxy: {
-                                            deceased: 'Details of the person acting on behalf of the claimant',
-                                            nonDeceased: 'Details of the person acting on behalf of the victim'
-                                        }
-                                    }
```

Comment:
Just making sure that this is fully intentional.

Reply:
Yes, that's deliberate. The rep details task always has the title "Your details", which makes sense as there is only one context a rep would ever see.